_includes/tutorials/console-consumer-producer-avro/confluent/markup/dev/answer-short.adoc

```diff
@@ -0,0 +1,13 @@
+With link:https://www.confluent.io/confluent-cloud/tryfree/[Confluent Cloud], you can use the Confluent Cloud CLI to produce and consume messages.
+
+Producer:
+
++++++
+<pre class="snippet"><code class="bash">{% include_raw tutorials/console-consumer-producer-avro/confluent/code/tutorial-steps/dev/console-producer-keys.sh %}</code></pre>
```

Comment:
One minor suggestion - the `console-producer-keys.sh` file has the location of schema file in the command so maybe add a quick note in the short answer that the path for the schema needs to change.  But I'll leave it up to you.

Reply:
@bbejeck done in latest commit